.github/workflows/packer-release.yml

```
@@ -0,0 +1,61 @@
+---
+name: packer-release
+
+on:
+  push:
+    tags:
+      - v*
+  workflow_dispatch:
+
+concurrency:
+  group: packer-release
+  cancel-in-progress: false
+
+env:
+  VAGRANT_CLOUD_TOKEN: ${{ secrets.VAGRANT_CLOUD_TOKEN }}
+  BOX_ORG: ${{ github.repository_owner }}
+
+jobs:
+  packer:
+    runs-on: macos-12
+    timeout-minutes: 60
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # NOTE: Even though ST2_VERSION and BOX_VERSION are the same in the Makefile is is arguably nice to see them here.
+      - name: Export ST2_VERSION and BOX_VERSION
+        run: |
+          echo "ST2_VERSION=$(curl --silent \
+            'https://api.github.com/repos/stackstorm/st2/releases/latest' \
+            | jq -r .tag_name | sed 's/^v//')" >> $GITHUB_ENV
+          # Turn tags like v3.8.0-20230119 into just 20230119
+          echo BOX_VERSION=$(echo "${{ github.ref_name }}" | cut -d'-' -f2) \
+            >> $GITHUB_ENV
+
+      - name: Packer validate
+        run: make validate
+
+      - name: Packer build
+        run: make build
+
+      - name: Create GH Release
+        uses: marvinpinto/action-automatic-releases@v1.2.1
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          automatic_release_tag: ${{ github.ref_name }}
+          prerelease: false
+          files: |
+            builds/st2_${{ github.ref_name }}.box
```

```suggestion
          files: |
            builds/st2_${{ github.ref_name }}.ova
```



Need to upload the `.ova` artifact to the GH Releases instead of `.box`
See https://github.com/StackStorm/packer-st2/releases

and so `.OVA` could be used manually in raw Virtualbox, while `.BOX` is for Vagrant/Cloud